### DISCUSSION TITLE
Fluent UI + TailwindCSS = ❤️

### DISCUSSION THREAD

**Discussion by dinbtechit** [microsoft/fluentui#29666] [Ideas] [answered]
Is there any plans to implement Microsoft Fluent UI in TailwindCSS

**Comment by layershifter** [comment, marked as answer]
No, Fluent UI React v9 relies on Griffel (https://griffel.js.org/) for styling.

However, nothing blocks you to use Tailwind on a consumer side as all components accept `className` prop.
